hack/update-swagger-spec.sh

```diff
@@ -36,6 +36,9 @@ function cleanup()
 
 trap cleanup EXIT SIGINT
 
+kube::log::status "Building apiserver"
+"${KUBE_ROOT}/hack/build-go.sh" cmd/kube-apiserver
```

Comment:
LGTM


we should have a --dump-swagger=VER flag, rather than running apiserver